osu.Game/Screens/Edit/Compose/Components/SelectionHandler.cs

```diff
@@ -59,6 +60,9 @@ public abstract partial class SelectionHandler<T> : CompositeDrawable, IKeyBindi
 
         public SelectionScaleHandler ScaleHandler { get; private set; }
 
+        [Resolved]
+        private OsuContextMenuContainer editorContextMenuContainer { get; set; }
```

Comment:
Would rather make this nullable than require it to be present on every test scene. I'm just worried it makes testing even harder than it already is with these components.

Reply:
Looks makes sense, maybe somewhere will needs the selection handler but without a container